### ISSUE TITLE
question about batchsize

### ISSUE THREAD

**Issue by liuxianshun** [akshaydudhane16/Burstormer#18]
In the paper, the author mentions setting the batch size to 4. However, in the forward function of the Burstormer network, the first step involves the operation burst = burst[0]. Doesn't this mean that, regardless of the batch size setting, the actual batch size becomes 1?

**Comment by rk764457656** [comment]
According my understanding, if  the setting of bs is 4 and more, the process of training is following:
for i in batch_size:
      output[i] = model(burst[i])
      computing loss
      loss += loss_i
update

**Comment by rk764457656** [comment]
from Author：
Burstormer is faster than BIPNet. In our settings, we keep precision 16 and deterministic True.
In released codes, each GPU serves a single burst (so 4 GPUs combinely make batch size=4). So, to improve training time, you can implement the network architecture such that you can increase batch size.

**Comment by liuxianshun** [comment]
> from Author： Burstormer is faster than BIPNet. In our settings, we keep precision 16 and deterministic True. In released codes, each GPU serves a single burst (so 4 GPUs combinely make batch size=4). So, to improve training time, you can implement the network architecture such that you can increase batch size.

thank you！

**Comment by atifkhanncl** [comment]
Hi, I don't understand the point 'In released codes, each GPU serves a single burst (so 4 GPUs combinedly make batch size=4)'. does it mean each gpu server a single frame in the burst? can you explain this clearer please?
